builders/build/historical-updates.md

```diff
@@ -854,3 +854,31 @@ This migration was executed at the following runtimes and blocks:
 | Moonbase Alpha |      RT1502      |    2112058    |
 
 For more information, you can review the [relative Nimbus PR](https://github.com/moonbeam-foundation/nimbus/pull/45){target=\_blank} and [Moonbeam PR on GitHub](https://github.com/moonbeam-foundation/moonbeam/pull/1400){target=\_blank}.
+
+### Foreign Assets Migration {: #foreign-assets-migration }
+
+A migration was applied to transition existing foreign assets to a new design that manages XCM derivative assets on Moonbeam through EVM smart contracts instead of the previous implementation using the `asset-manager` and `pallet-asset` pallets.
```

Comment:
On this page we typically spell out pallet names, so I'd recommend removing them from backticks and sticking with that format for consistency